composer.json

```diff
@@ -26,12 +26,12 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.1|^8.0"
     },
     "require-dev": {
         "phpstan/phpstan": "^0.12.37",
         "phpunit/phpunit": "^7.3 || ^8.0 || ^9.0",
-        "vimeo/psalm": "^3.13"
```

Comment:
as it's just a dev dependency I do not see impediments to bump it `^4.0`

do I miss anything?

Reply:
Agreed - I have bumped it to ^4.3, which is current minor release and is still compatible with PHP 7.1.